PhpAmqpLib/Connection/AMQPConnectionConfig.php

```diff
@@ -333,6 +333,10 @@ public function isSecure(): bool
     public function setIsSecure(bool $isSecure): void
     {
         $this->isSecure = $isSecure;
+
+        if ($this->isSecure) {
+            $this->setNetworkProtocol("ssl");
```

Comment:
This should be defined as constant. BTW, there are more secure protocol, do You think `ssl` is most common?

Reply:
At this time, we should make `tls` the default. What I'm wondering, however, is if it can be considered a breaking change requiring a 4.0 release 🤔

Reply:
I think just throw an Exception is better then, if it is secure but network protocol is not ssl or tls. At least they will know what to set.

The exception is not catching this at StreamIO